src/test/ui/borrowck/issue-64453.rs

```diff
@@ -13,7 +11,6 @@ fn set_editor(_: Value) {}
 
 fn main() {
     let settings_data = from_string(settings_dir);
-    //~^ ERROR cannot move out of static item `settings_dir` [E0507]
```

Comment:
It's unfortunate that this error no longer occurs. I believe it's simply because the compiler stops working before the move checker runs?